### DISCUSSION TITLE
SSP compliance for exported .SSD files?

### DISCUSSION THREAD

**Discussion by Fredrik1Haider** [OpenModelica/OMSimulator#1186] [Q&A] [answered]
Hi, in my naivety I was just wondering for the reason of adding additional elements (as per the schema /master/schema/oms.xsd) to the SSD when exporting it from OMS? Since this breaks the SSP compliance (checking the .ssd with the associated schema fails)?

**Comment by lochel** [comment]
The additional elements are part of vendor specific annotations and shouldn't break the ssp compliance. However, maybe we are doing something slightly wrong. @arun3688 can you check that? The reason for us introducing those elements is that the SSP standard group couldn't agree on a way to describe solver related model information and it was removed last minute form the standard (back then).

**Comment by arun3688** [comment, marked as answer]
@Fredrik1Haider As lennart said the additional elements are part of vendor specific annotation  and these additional elements belong to `oms:SimulationInformation` and they are valid according to ssp specification

**Reply by Fredrik1Haider** [reply]
Okay, then I will have to look over my methodology for compliance checking. Thank you for a quick response!
